src/pages/en/guides/publish-to-npm.md

```diff
@@ -6,37 +6,40 @@ description: Learn how to publish Astro components to NPM
 
 Building a new Astro component? **Publish it to [npm!](https://npmjs.com/)**
 
-Publishing a component is a great way to reuse work across your team, your company, or the entire world. Astro components can be published to and installed from npm, just like any other JavaScript package.
+Publishing an Astro component is easy and a great way to reuse work across your projects, with others, and with the wider Astro-community at large. Astro components can be published directly to and installed from NPM, just like any other JavaScript package.
```

Comment:
This doesn't really make sense. I suggest changing "reuse" to "share".

Reply:
I see where you're coming from! I think the idea is that if you make a component once, then publish it, you (and other people) can USE it all over the place! Even in other projects!

Since we don't hae the word "share" already in this paragraph, I can see "share" being a good word choice here!

Reply:
(oooh, I am probably gonna have to vote for taking out "is easy" -- that's typically a no-no in docs! You can't assume what's easy for people, and it makes them feel bad if it wasn't easy for them!)

Reply:
In this case you are not sharing your component within your project you are reusing it more than once, sharing in this context is very loose and fast with the language

Reply:
+1 for removing “easy” and using “share” from me. I also get where the “reuse” comes from but “reuse with others” seems off to me.

Reply:
THIS SOUNDS LIKE A CHALLENGE FOR ME TO FIT IN BOTH WORDS.  ;)